content/cucumber/anti-patterns.md

```diff
@@ -0,0 +1,135 @@
+---
+source: https://github.com/cucumber/cucumber/wiki/Feature-Coupled-Step-Definitions-(Antipattern)/
+source: https://github.com/cucumber/cucumber/wiki/Conjunction-Steps-(Antipattern)/
+source: https://stackoverflow.com/questions/22696646/how-to-call-a-step-from-another-step-in-cucumber-jvm
+source: https://groups.google.com/forum/#!searchin/cukes/jvm$20steps$20programming/cukes/DzE_kGZx94I/5rf__N31qvAJ
```

Comment:
A question about the YAML front-matter: Will repeating keys overwrite the last one in the system we're using?

In [the Jekyll YAML Frontmatter documentation](https://jekyllrb.com/docs/frontmatter/), a mention is made of `categories`, which can be specified [as a YAML list](https://en.wikipedia.org/wiki/YAML#Basic_components).

_If_ we're not programmatically using the `source` key, perhaps we can change it to a YAML list `sources` (plural):

```yaml
sources:
  - https://github.com/cucumber/cucumber/wiki/Feature-Coupled-Step-Definitions-(Antipattern)/
  - https://github.com/cucumber/cucumber/wiki/Conjunction-Steps-(Antipattern)/
  - https://stackoverflow.com/questions/22696646/how-to-call-a-step-from-another-step-in-cucumber-jvm
  - https://groups.google.com/forum/#!searchin/cukes/jvm$20steps$20programming/cukes/DzE_kGZx94I/5rf__N31qvAJ
```

Reply:
@olleolleolle Tbh I don't actually know. I figured I'd list the sources that I'd merged into the documents. I'd be happy to do so differently if needed...

Reply:
The `sources` front matter is not programatically used at all - it was just a way of keeping track of which page the new pages were coming from.

Might make more sense to leave it as `source` but a list, and then test it for list / url if we ever use it?

Reply:
Thanks, this makes sense to me.

This note of mine - A low-priority thing and “for later”, not blocking this change at all.

Reply:
Leaving this as is for now.